dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/DatasetServiceBean.java

```diff
@@ -674,6 +674,28 @@ public void obtainPersistentIdentifiersForDatafiles(Dataset dataset) {
         }
     }
 
+    /**
+     *
+     * @param dataset
```

Comment:
The params comment should be used only if you explain the context. For example take a look at apache method.
```
@param str  the String to check, may be null

StringUtils.isEmpty(str)
```
If it doesn't bring much value I would just remove it.